Sources/Views/PageLayout.swift

```diff
@@ -282,10 +282,10 @@ Prism.languages.swift = Prism.languages.extend('clike', {
       }
     }
   },
-  'keyword': /\b(?:actor|as|associativity|async|await|break|case|catch|class|continue|convenience|default|defer|deinit|didSet|do|dynamic(?:Type)?|else|enum|extension|fallthrough|final|for|func|get|guard|if|import|in|infix|init|inout|internal|is|lazy|left|let|mutating|new|none|nonisolated|nonmutating|operator|optional|override|postfix|precedence|prefix|private|protocol|public|repeat|required|rethrows|return|right|safe|self|Self|set|some|static|struct|subscript|super|switch|throws?|try|Type|typealias|unowned|unsafe|var|weak|where|while|willSet|__(?:COLUMN__|FILE__|FUNCTION__|LINE__))\b/,
+  'keyword': /\b(?:actor|Any|as|associativity|associatedtype|async|await|break|case|catch|class|continue|convenience|default|defer|deinit|didSet|do|dynamic(?:Type)?|else|enum|extension|fallthrough|false|fileprivate|final|for|func|get|guard|if|import|in|indirect|infix|init|inout|internal|is|lazy|left|let|mutating|new|nil|none|nonisolated|nonmutating|open|operator|optional|override|postfix|precedence|precedencegroup|prefix|private|protocol|Protocol|public|repeat|required|rethrows|return|right|safe|self|Self|set|some|static|struct|subscript|super|switch|throws?|true|try|Type|typealias|unowned|unsafe|var|weak|where|while|willSet|_(?:modify|read)?|__(?:COLUMN__|FILE__|FUNCTION__|LINE__)|#(?:available|colorLiteral|column|dsohandle|elseif|else|endif|error|fileID|fileLiteral|filePath|file|function|if|imageLiteral|keyPath|line|selector|sourceLocation|warning))\b/,
```

Comment:
We were missing `associatedtype` somehow, so added it along with a bunch of others.